src/ui/components/filters/facetscomponent.js

```diff
@@ -251,10 +245,30 @@ export default class FacetsComponent extends Component {
     return super.setState({
       ...data,
       filters: Facet.fromCore(processedFacets),
+      filterOptionsConfigs: this._getFilterOptionsConfigs(processedFacets),
       isNoResults: data.resultsContext === ResultsContext.NO_RESULTS
     });
   }
 
+  /**
+   * Gets filter options from transformedFacets and puts them in an object
+   * keyed by fieldId
+   *
+   * @param {DisplayableFacet | FilterOptionsConfig} transformedFacets a union of the
+   * DisplayableFacet model from ansers-core, and the config options of the FilterOptionsConfig
+   * @returns {Object} config options of the FilterOptionsConfig keyed by filterId
+   */
+  _getFilterOptionsConfigs (transformedFacets) {
+    return transformedFacets.reduce((acc, currentFacet) => {
+      const filterOptions = Object.assign({}, currentFacet);
+      // Don't support options when extracting filterOptions from transformedFacets
+      // because the core library DisplayableFacet isn't compatible with FilterOptionsConfig.options
```

Comment:
What does this mean exactly? Are people unable to configure the options when using `transformFacets`?

Reply:
I've updated the comment to give some extra context